packages/astro/src/core/errors/errors-data.ts

```diff
@@ -397,6 +397,18 @@ export const NoAdapterInstalled = {
 	hint: 'See https://docs.astro.build/en/guides/server-side-rendering/ for more information.',
 } satisfies ErrorData;
 /**
+ * @docs
+ * @see
+ * - [Server-side Rendering](https://docs.astro.build/en/guides/server-side-rendering/)
+ * @description
+ * To use server islands, the same constraints exist as for sever-side rendering, so an adapter is needed.
+ */
+export const NoAdapterInstalledServerIslands = {
+	name: 'NoAdapterInstalledServerIslands',
+	title: 'Cannot use Server Islands without an adapter.',
+	message: `Cannot use server islands without an adapter. Please install and configure the appropriate server adapter for your final deployment.`,
+	hint: 'See https://docs.astro.build/en/guides/server-side-rendering/ for more information.',
+} satisfies ErrorData;/**
```

Comment:
```suggestion
} satisfies ErrorData;
/**
```
the formatting looks weird here?